code/upaya/Gemfile

```diff
@@ -55,7 +55,7 @@ group :development do
   gem 'aws-sdk', '~> 2.0'
   gem 'better_errors'
   gem 'derailed'
-  gem 'binding_of_caller', platforms: [:mri_21]
+  gem 'binding_of_caller', platforms: [:mri_23]
```

Comment:
I concur, I was just making quick fixes without thinking forward. I even hesitate to fix the version in the gemfile unless it's needed for compatility / ease of dev setup (which I believe it is in this case). Usually I would leave that up to the dev in a .ruby-version.example file. Let's remove the platform statement and set 2.3.0 at the top of them Gemfile as you suggest.